spec/models/series_spec.rb

```diff
@@ -26,63 +26,63 @@
     )
   end
 
-  let!(:series) do
-    Series.create!(
-      name: 'foo bar',
-      description: 'Amig Goswami é uma das maiores autoridades mundiais em
-        Psicologia Quântica. Nesta palestra, ele faz um contraponto entre a
-        Psicologia tradicional e a Quântica. Ele sua fala ele mostra que o
-        modelo comportamental',
-      category: category,
-      image_url: 'http://foo'
-    )
-  end
-
-  let!(:series2) do
-    Series.create!(
-      name: 'foo bar',
-      description: 'Amig Goswami é uma das maiores autoridades mundiais em
-        Psicologia Quântica. Nesta palestra, ele faz um contraponto entre a
-        Psicologia tradicional e a Quântica. Ele sua fala ele mostra que o
-        modelo comportamental',
-      category: category
-    )
-  end
-
-  let!(:video) do
-    Video.create!(
-      name: 'foo bar',
-      description: 'Amig Goswami é uma das maiores autoridades mundiais em
-        Psicologia Quântica. Nesta palestra, ele faz um contraponto entre a
-        Psicologia tradicional e a Quântica. Ele sua fala ele mostra que o
-        modelo comportamental',
-      vimeo_id: '163721649',
-      series: series2,
-      image_url: 'http://foobar.png'
-    )
-  end
-
   describe 'associations' do
-    it { expect(subject).to belong_to(:category) }
-    it { expect(subject).to have_many(:videos) }
+    it { is_expected.to belong_to(:category) }
+    it { is_expected.to have_many(:videos) }
   end
 
   describe 'validations' do
-    it { expect(subject).to validate_presence_of(:category) }
-    it { expect(subject).to validate_presence_of(:name) }
-    it { expect(subject).to validate_presence_of(:description) }
+    it { is_expected.to validate_presence_of(:category) }
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:description) }
   end
 
-  describe 'get image url' do
+  describe '.image' do
```

Comment:
This should be `#image` instead of `.image`. We use `#method_name` when we're referring to instance methods and `.method_name` for class methods.